### PR TITLE
Ghostdrone Fix

### LINKED ISSUE #8306: Thindoors are one-way for ghost critters
### Labels

[MINOR]

### Description

Ghost critters can squeeze past the outer side of a thindoor, but they can't squeeze past the inner side

### Steps to reproduce

1. Become a ghost critter/drone
2. Squeeze through a thindoor (like the ones present on many department's service counters) from the outer side of it
3. Be unable to squeeze back from the inner side of the thindoor

### Expected Behavior

Ghost critters should be able to squeeze past both sides of a thindoor freely

### Additional Information & Screenshots

image

### PR DESCRIPTION
**Pull request by Laboredih123**
[BUG - MINOR]

## About the PR 
Fixes #8306

## Why's this needed? 
Bug no bueno and I don't like getting stuck

### CODE REVIEW

**Comment by Zonespace27** [comment]
#10000 🎊

File: code/obj/machinery/door/airlock.dm [hunk 1]
```diff
@@ -619,7 +619,7 @@ Airlock index -> wire color are { 9, 4, 6, 7, 5, 8, 1, 2, 3 }.
 					UNCROSS_BUMP_CHECK(mover)
 					return
 				animate_door_squeeze(mover)
-				. = 1 // they can pass through a closed door
+				return 1 // they can pass through a closed door
```
Comment (Zonespace27) [inline code/obj/machinery/door/airlock.dm:622]: True/false defines aaaaaa

**Comment by ThatFiveGuys** [comment]
This fixes the issue of ghost critters not being able to squeeze through right?

**Comment by Laboredih123** [comment]
yah

File: code/obj/machinery/door/airlock.dm (hunk 1 above)
Comment (Laboredih123) [reply code/obj/machinery/door/airlock.dm:622]: a milestone isnt without its mistakes sadly

**Comment by Two-Braids** [comment]
no more getting stuck! Woo!

**Pull request merged**